src/test/java/assignment1/Assignment1Test.java

```diff
@@ -29,7 +29,29 @@ public void verifyNotEnoughArguements() {
   }
 
   @Test
-  @DisplayName("SI Hashmap is being created properly")
+  @DisplayName("Exception thrown if invalid value for price is passed")
```

Comment:
create a testcase where price given is in decimal (valid and invalid both)

Reply:
Resolved.